libs/ledger-live-common/src/families/tron/synchronization.ts

```diff
@@ -167,6 +167,9 @@ export const getAccountShape: GetAccountShape<TronAccount> = async (
       return sub;
     }),
   );
+
+  const mergedSubAccounts = mergeSubAccounts(subAccounts, initialAccount?.subAccounts || []);
```

Comment:
Can you please add why we do it here ? (eg. not returned by backend, needed for swap, ticket number)